src/plugins/validation/2and3/semantic-validators/paths-ibm.js

```diff
@@ -134,6 +135,31 @@ module.exports.validate = function({ resolvedSpec }, config) {
           });
         }
       });
+    } else {
+      // in the else block because usage of paths_case_convention is mutually
+      // exclusive with usage of config.snake_case_only since it is overlapping
+      // functionality
+      if (config.paths_case_convention) {
```

Comment:
This `if` statement is not really necessary. It seems weird to not check if a property is defined before referencing it, but [there is code](https://github.com/IBM/openapi-validator/blob/8152d06e184963abe101702dd69250735dfa9b90/src/cli-validator/utils/processConfiguration.js#L123-L146) that guarantees all configuration options are set by the time it reaches the individual validations. So, this will always be true.

If you prefer to leave it in, that's fine with me but I think it's a somewhat redundant check.

Reply:
Originally I did not have it in, but during the automated tests there were crashes because this was not defined.  My guess is that the test harness somehow bypasses this check you mentioned?

Reply:
Ah, that is correct. The test framework runs validations individually and never touches the shared utility code. I've been slowly updating the tests to be less fragile with respect to the config object by bringing in the entire "defaults" object and using that, rather than defining a config object for each each individual test. I just hadn't gotten around to the `paths-ibm` test yet. I'll merge this as is and fix that at a later time